README.md

```diff
@@ -54,6 +54,17 @@ Include the default recipe in a run list, to get `php`. By default `php` is inst
 
 This recipe installs PHP from packages.
 
+### community_package
+
+This recipe intalls PHP from one of two available community package repositories, depending on platform family. This provides the ability to install PHP versions that are no provided by the official distro repositories.
+
+Set `node['php']['install_method'] = community_package` to use these repositories.
+
+Please see `test/cookbooks/test/recipes/community.rb` for an example of how to use attributes to install the desired version of PHP & its supporting packages, and please refer to the documentation on these community repositories:
+
+- RHEL/Amazon - [Remi’s RPM repository](https://rpms.remirepo.net)
```

Comment:
Do we want to care about IUS repos for RHEL? They currently only work on RHEL7. We use it FWIW but we may switch to Remi's at some point.

Reply:
@ramereth I picked Remi’s b/c it appeared to be on the same level of “use this” from various search results as the APT PPA, is still kept up-to-date, and offers a broad selection of PHP versions for various flavors of EL7 & EL8.

I didn't come across the IUS repos in my search! Looking through the [IUS repo](https://repo.ius.io), seems its not as up-to-date as Remi’s; it doesn’t currently offer PHP >7.4. Pretty cool project though.